skfem/element/element_hex/element_hex_s2.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+
+from ..element_h1 import ElementH1
+from ...mesh.mesh3d import MeshHex
+
+
+class ElementHexS2(ElementH1):
+    nodal_dofs = 1
+    edge_dofs = 1
+    dim = 3
+    maxdeg = 3
```

Comment:
Is maxdeg = 3?  I would have thought it was 4, like `ElementHex2`; the eight `nodal_dofs` are quartic.

Reply:
Increased this to 4. Also decreased to 3 in `ElementQuadS2`.